packages/theme-addresses/theme-addresses.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * CountryProvinceSelector Constructor
+ * @param {Object} countryOptions Country JSON object
+ */
+export function CountryProvinceSelector(countryOptions) {
+  this.countryOptions = countryOptions
+}
+
+/**
+ * Helper function that builds the province selector
+ */
+var buildProvince = function (provinceNodeElement, provinces) {
```

Comment:
I think I remember seeing something ages ago, but it's mostly been replaced by [Shopify ESLint](https://github.com/Shopify/eslint-plugin-shopify) and [stylelint](https://github.com/Shopify/stylelint-config-shopify). However I'm not using Shopify Eslint for the core files of this repo (I am for tests) because the ES5 config is broken. Would be worth improving though? 😅

Reply:
I don't have to use function assignment but using `(function fnName() {})` is a self execution of the function. You will see this pattern commonly used for declaring a set of library methods on the javascript window scope like jQuery, for example
```js
(function() {
   ...
   this.jquery = jQuery    // This binds window.jquery to jQuery instance
})(window)
```
My methods are just helper methods that should stay inside this module closure